DATA/production/qc-workflow.sh

```diff
@@ -11,13 +11,13 @@ elif [[ -z $QC_JSON_FROM_OUTSIDE ]]; then
   if [[ $EPNSYNCMODE == 1 || "0$GEN_TOPO_LOAD_QC_JSON_FROM_CONSUL" == "01" ]]; then
     [[ -z "$QC_JSON_TPC" ]] && QC_JSON_TPC=consul://o2/components/qc/ANY/any/tpc-full-qcmn
     [[ -z "$QC_JSON_ITS" ]] && QC_JSON_ITS=consul://o2/components/qc/ANY/any/its-qcmn-epn-full
-    [[ -z "$QC_JSON_MFT" ]] && QC_JSON_MFT=consul://o2/components/qc/ANY/any/mft-full-qcmn
```

Comment:
Ah right, because when you set `QC_JSON_MFT` here then the check below `[[ -z $QC_JSON_MFT ]]` is false and therefore you don't use the mft-track-full-qcmn even if MFT_RECO is enabled